crates/pessimistic-proof-test-suite/tests/snapshots/cycle_tracker__s01_be002.snap

```diff
@@ -1,46 +1,42 @@
 ---
 source: crates/pessimistic-proof-test-suite/tests/cycle-tracker.rs
 expression: stats
-snapshot_kind: text
 ---
-gas: 4204527
-opcode counts (1264005 total instructions):
-    323610 add
-    189424 sw
-    169319 lw
-    108648 lbu
-     80845 sb
-     54548 xor
-     47317 beq
-     47279 bne
-     34061 bltu
-     29746 sltu
-     26054 jalr
-     25992 and
-     22804 sub
-     20137 or
-     19591 sll
-     19198 srl
-     13034 auipc
-      7568 sra
-      7533 lb
-      4967 mul
-      4202 mulhu
-      2970 jal
-      2106 ecall
-      2069 bgeu
-       895 sh
-        44 lh
-        34 lhu
-         6 blt
-         4 bge
-syscall counts (2106 total syscall instructions):
-    1024 secp256k1_double
-     525 keccak_permute
-     521 secp256k1_add
-       8 commit
-       8 commit_deferred_proofs
-       8 hint_len
-       8 hint_read
-       3 write
-       1 halt
+gas: 25718188
+opcode counts (23846614 total instructions):
```

Comment:
This is a huge diff in terms of `opcode` and `gas`, what do you think @iljakuklic ?